pact/marmalade.repl

```diff
@@ -1,2 +1,348 @@
-(load "./policy-manager/policy-manager.repl")
-(typecheck "marmalade-v2.ledger")
```

Comment:
Why aren't we typechecking the ledger anymore?

Reply:
Wanted to keep typecheck of the contract in the corresponding repl, but mistakenly put `policy-manager` typecheck instead of `ledger`. Fixed